public/pages/Reindex/container/Reindex/Reindex.tsx

```diff
@@ -661,31 +682,31 @@ class Reindex extends Component<ReindexProps, ReindexState> {
           </EuiFlexGroup>
         </ContentPanel>
 
-        <EuiSpacer />
+          <EuiSpacer />
 
-        <ContentPanel title={advanceTitle} noExtraPadding>
-          {advancedSettingsOpen && (
-            <>
-              <EuiSpacer size="s" />
-              <ReindexAdvancedOptions
-                slices={slices}
-                onSlicesChange={this.onSliceChange}
-                sliceErr={this.state.sliceError}
-                getAllPipelines={this.getAllPipelines}
-                selectedPipelines={this.state.selectedPipelines}
-                onSelectedPipelinesChange={this.onPipelineChange}
-                ignoreConflicts={ignoreConflicts}
-                onIgnoreConflictsChange={this.onIgnoreConflictsChange}
-                reindexUniqueDocuments={reindexUniqueDocuments}
-                onReindexUniqueDocumentsChange={this.onReindexUniqueDocuments}
-              />
-              <NotificationConfig ref={(ref) => (this.notificationRef = ref)} actionType={ActionType.REINDEX} />
-              <EuiSpacer size="s" />
-            </>
-          )}
-        </ContentPanel>
+          <ContentPanel title={advanceTitle} noExtraPadding>
+            {advancedSettingsOpen && (
+              <>
+                <EuiSpacer size="s" />
```

Comment:
The indentation doesn't match the rest; should others be corrected or these?

Reply:
Fixed